pipeline/common/build.gradle.kts

```diff
@@ -22,4 +31,99 @@ dependencies {
 
 detekt {
     config = files("detekt-config.yml")
+}
+
+tasks.dokka {
+    outputFormat = "html"
+    outputDirectory = "$buildDir/javadoc"
+}
+
+val sourcesJar by tasks.creating(Jar::class) {
+    archiveClassifier.set("sources")
+    from(sourceSets.getByName("main").allSource)
+}
+
+val dokkaJar by tasks.creating(Jar::class) {
+    description = "Assembles Kotlin docs with Dokka"
+    classifier = "javadoc"
+    from(tasks.dokka)
+}
+
+
+
+publishing {
+    publications {
+        create<MavenPublication>("pipeline-common") {
+            artifactId = "pipeline-common"
+            groupId = pubGroup
+            version = "0.1.0"
+            from(components["java"])
+
+            // include sources jar
+            artifact(sourcesJar)
+
+            // include kdoc jar
+            artifact(dokkaJar)
+
+            pom {
+                name.set("$pubGroup:pipeline-common")
+                description.set("Easily write pipeline plugins for the Hyperion logging framework.")
+                url.set("https://github.com/SERG-Delft/hyperion")
+
+
+                licenses {
+                    license {
+                        name.set("The Apache License, Version 2.0")
+                        url.set("http://www.apache.org/licenses/LICENSE-2.0.txt")
+                    }
+                }
+
+                developers {
+                    developer {
+                        id.set("Hyperion authors")
+                    }
+                }
+
+                scm {
+                    url.set("https://github.com/SERG-Delft/hyperion")
+                }
+            }
+        }
+    }
+
+    repositories {
+        maven {
+            // update this url to correct repo
+            url = uri("https://oss.sonatype.org/service/local/staging/deploy/maven2")
+            credentials {
+                // Here is the previous account registered in issues. sonatype. org.
+                username = project.properties["sonatypeUsername"].toString()
+                password = project.properties["sonatypePassword"].toString()
+            }
+        }
+    }
+}
+
+// bintray {
+//     user = project.findProperty("bintrayUser").toString()
+//     key = project.findProperty("bintrayKey").toString()
+//     publish = true
+//     setPublications("pipeline-common")
+//
+//     pkg.apply {
+//         repo = "hyperion"
+//         name = "$pubGroup:pipeline-common"
+//         setLicenses("Apache-2.0")
+//         userOrg = "serg-tudelft"
+//         vcsUrl = "https://github.com/SERG-Delft/hyperion.git"
+//         version.apply {
+//             name = "0.1.0"
+//             desc = "0.1.0"
+//             released = Date().toString()
+//         }
+//     }
+// }
```

Comment:
Should this commented section be removed?

Reply:
haha yes I suck, I will delete it